### DISCUSSION TITLE
Maintainer wanted

### DISCUSSION THREAD

**Discussion by jschlyter** [hass-chargeamps/hass-chargeamps#67] [Announcements]
As neither @fredriklj or myself are active Charge Amps users this component might benefit from a more active maintainer. Anyone interested?

Ping @fredrike

**Comment by fredrike** [comment]
I do not have a chargeAmps charger anymore.

**Comment by jschlyter** [comment]
ping @fekberg ?

**Comment by fekberg** [comment]
@jschlyter I do have two Halo chargers and will definitely help out if the api changes, or if there are bugs that I run in to. But I don't have time to actively maintain the repo. Happy to try and help out when I can.

**Comment by jschlyter** [comment]
@fekberg That is most likely enough for now. I've integrated the ChargeAmps client directly into the component so future releases are easier to maintain. The standalone `python-chargeamps` will most likely die.

I'm happy to keep the repo here for now and will ask the community for help if there are any issues.

**Comment by jschlyter** [comment]
@JohnLindahlTech will take care of the integration and once I see everything is moving forward I'll let him take the helm for good.
